os/emscripten/pre.js

```diff
@@ -65,10 +65,10 @@ Module.preRun.push(function() {
     }
 
     window.openttd_server_list = function() {
-        add_server = Module.cwrap("em_openttd_add_server", null, ["string", "number"]);
+        add_server = Module.cwrap("em_openttd_add_server", null, ["string"]);
 
         /* Add servers that support WebSocket here. Example:
-         *  add_server("localhost", 3979); */
+         *  add_server("localhost"); */
```

Comment:
Would it make sense to also add a line mentioning the <hostname>:<port> format, e.g.: `* add_server("localhost:3979");`. Although I'm not assuming that is going to work.